spec/lib/fields/virtual_field_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+require "administrate/field/base"
+
+module Administrate
+  module Field
+    class VirtualField < Field::Base
```

Comment:
I wonder if we can find an example in the dashboard where we can define this type of virtual field without having to drop it here?

Reply:
What else is needed here? Documentation, maybe?
Or would an approach using an option like `virtual_field: true` to explicitly indicate virtual fields work better?

Reply:
Answering on the main thread :slightly_smiling_face: